contracts/Bridge/token/XVS.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import { TokenController } from "./TokenController.sol";
+
+contract XVS is ERC20, TokenController {
```

Comment:
Have you considered the association instead of the inheritance for the relationship XVS-TokenController? This way, we could keep the XVS contract not upgradable, but with a setter function for the token controller. I understand one downside would be the extra gas in the transactions. Well, just an idea to think about